### DISCUSSION TITLE
GitHub Support ticket been unattended for almost 60 days

### DISCUSSION THREAD

**Discussion by ghost** [community/community#127884] [New to GitHub]
So i've had this ticket opened for 60 days related to github education, because they got a school e-mail domain wrong and I can't join the program. Is it normal that it takes that long?
![image](https://github.com/community/community/assets/167018979/726d0a0f-d1c1-4959-9ef9-aef205a9f429)

**Comment by davevad93** [comment]
Hi @ibmathdev ,

GitHub support may take a long time to attend, some users had to wait for months. I guess it's due to a lot of pending tickets so be patient and wait.
